presto-orc/src/main/java/com/facebook/presto/orc/StripeReader.java

```diff
@@ -121,8 +124,9 @@ public StripeReader(
         this.metadataReader = requireNonNull(metadataReader, "metadataReader is null");
         this.writeValidation = requireNonNull(writeValidation, "writeValidation is null");
         this.stripeMetadataSource = requireNonNull(stripeMetadataSource, "stripeMetadataSource is null");
-        this.cacheable = requireNonNull(cacheable, "hiveFileContext is null");
+        this.cacheable = requireNonNull(cacheable, "cacheable is null");
```

Comment:
requireNonNull can be removed, as cacheable is a boolean(primitive) and will never be null.

Reply:
Oh, right. Thanks for pointing it out. Done.